src/App.js

```diff
@@ -1,43 +1,40 @@
-import logo from './logo.svg';
-import './App.css';
-import TodoList from './todoList';
+import React, { useState, useEffect } from 'react';
 import AddTodoForm from './AddTodoForm';
-import { useState } from 'react';
+import TodoList from './TodoList';
 
-/* 
-const todoList = [{id:1,title:'React-lesson-1',content:'Project Setup',bookName:'Road to React'},
-                    {id:2,title:'React-lesson-2',content:'JSX in React',bookName:'Road to React'},
-                    {id:3, title:'React-lesson-3',content:'React components',bookName:'Road to React'}
-                  ]; */
+const useSemiPersistentState = ()=>{
+  const [todoList, setTodoList]= useState([]);
```

Comment:
Looks like line 11 needs to be included in this line. We want to grab local storage on initial load up. Best place to do that is when the initial state is being initialized. 

```suggestion
  const [todoList, setTodoList] = useState(
    JSON.parse(localStorage.getItem('savedTodoList')) || []
  );```